package.json

```
@@ -62,7 +62,7 @@
     "aria-query": "^4.0.1",
     "array-includes": "^3.0.3",
     "ast-types-flow": "^0.0.7",
-    "axe-core": "^3.4.0",
+    "axe-core": "3.4.1",
```

```suggestion
    "axe-core": "=3.4.1",
```
this conveys it's intentionally pegged